kodilla-hibernate/src/main/java/com/kodilla/hibernate/tasklist/TaskList.java

```diff
@@ -0,0 +1,61 @@
+package com.kodilla.hibernate.tasklist;
+
+import com.kodilla.hibernate.task.Task;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "TASKLISTS")
+public final class TaskList {
+    private int id;
+    private String listName;
+    private String description;
+    private List<TaskList> taskList;
+
+    public TaskList() {
+    }
+
+    public TaskList(String listName, String description) {
+        this.listName = listName;
+        this.description = description;
+        this.taskList = new ArrayList<>();
```

Comment:
Zmieniłeś przenosząc inicjalizację do konstruktora - w sumie nie o to mi chodziło. To pole jest w ogóle nieużywane - nie ma do niego żadnego gettera.